config/crd/bases/cluster.kubernetes-fleet.io_memberclusters.yaml

```diff
@@ -100,6 +100,9 @@ spec:
                   name:
                     description: Name of the object being referenced.
                     type: string
+                    x-kubernetes-validations:
```

Comment:
can we add the marker in the api itself? the crd will be regenerated each time.

Could you please add a e2e test?

Reply:
https://github.com/Azure/fleet/tree/main/test/e2e e2e tests